src/main/java/com/vitalii/twenty/methods/TwentyMethods.java

```diff
@@ -17,31 +21,41 @@ public static char[] modifyCharArrayToIntArray(int[] ints) {
         return chars;
     }
 
-    //3 приминает 2 инта, а и б, возвращает большее из этих 2х чисел
+    /**
+     * 3 принимает 2 инта, а и б, возвращает большее из этих 2х чисел.
+     */
     public static int returnMaxOfTwoInts(int a, int b) {
         if (a > b) {
             return a;
         }
         return b;
     }
 
-    //3.1 приминает 2 инта, а и б, возвращает большее из этих 2х чисел
+    /**
+     * 3.1 принимает 2 инта, а и б, возвращает большее из этих 2х чисел
+     */
     public static int returnMaxOfTwoIntsWithMath(int c, int d) {
         return Math.max(c, d);
     }
 
-    //3.2 приминает 2 инта, а и б, возвращает большее из этих 2х чисел
+    /**
+     * 3.2 принимает 2 инта, а и б, возвращает большее из этих 2х чисел
+     */
     public static int returnMaxOfTwoIntsWithThernary(int a, int b) {
         return (a > b) ? a : b;
     }
 
-    //4 принимает 3 инта, возвращает большее из них
+    /**
+     * 4 принимает 3 инта, возвращает большее из них
+     */
     public static int returnMaxOfThreeInts(int a, int b, int c) {
         int tempMax = returnMaxOfTwoInts(a, b);
         return returnMaxOfTwoInts(tempMax, c);
     }
 
-    //5 приминает 5 интов, возвращает большее из них
+    /**
+     * 5 принимает 5 интов, возвращает большее из них
+     */
     public static int returnMaxOfFiveIntsWithVarargs(int... nums) {
```

Comment:
We must be sure that get 5 ints as arguments. Rewrite this function.

Reply:
check array length to be exactly 5

Reply:
fixed